torch/csrc/jit/python/python_ivalue.h

```diff
@@ -42,6 +42,19 @@ struct C10_EXPORT ConcretePyObjectHolder final : PyObjectHolder {
     return py::str(py_obj_);
   }
 
+  std::vector<at::Tensor> extractTensors() override {
+    pybind11::gil_scoped_acquire ag;
+    // We could implement this entirely in C++ via pybind11 but it turns out to
+    // be substantially slower. Namely, the total time taken by markCompleted on
+    // a CUDAFuture is 21.5us with this implementation, but goes up to 58.7us
+    // when using C++. The reason is unclear.
+    // FIXME There probably is a better package where this code could reside?
```

Comment:
Since in another comment you mentioned dependencies between targets, I'm even more convinced that putting this Python helper function in the RPC module is a bad idea, because it basically means that JIT now has a dependency on RPC. The correct way to do this is to put this Python helper somewhere in a JIT module. Do you know what a good place would be?

Reply:
I am not sure if there is a better place to put this, but I agree with you, this is certainly better than having this in RPC, and I don't see any issue for the current version